Prithvi011-021/020-ExplorerMoment.js

```diff
@@ -0,0 +1,90 @@
+//Explore momentjs library and print a date in
+//all the available formats provided by momentjs.
+
+//This is UTC Formate 2021-08-11T11:36:28+05:30
+//Moment js is library which helps parsing, validating, manipulating and displaying the date and time in js
+const moment = require('moment')
+
+// it will print currunt time in UTC formate/now it can also support empy array and object
+const currentDate = moment();
+
+//// it will print in string formate that can be understand able
+console.log(currentDate.toString());
+
+/// When we creating the string from momentjs we have to first check it matches
+//ISO 8601  the  formats, we then check if the string matches the RFC 2822 Date time other wise it will generate a fall back of new Date(string)
+//String
+const day = moment("6 Mar 2017 21:22:23 GMT").toString();
+console.log(day);
+
+//String + Formate
+
+const formate1 = moment().format("12-25-1995", "MM-DD-YYYY");
+console.log(formate1);
+const formate2 = moment().format('YYYY/MM/DD');
+console.log(formate2);
+
+// If i parsing the date and time it will show me unexpected error which can 
+//not be identify by normal script so we have to use 'use strict' to identifined error
+const formate3 = moment('24/12/2020 09:15:00', "DD MM YYYY hh:mm:ss");
+//If i pass any date formate YYYY -> 24/12/2020 it will show only 2015/09/24
+console.log(formate3)
+
+//Formates Dates
+console.log(moment().format('YYYY Do MMMM, h:mm:ss a'))
+moment().format('MMMM Do YYYY, h:mm:ss a'); // August 11th 2021, 12:36:20 pm if I write MMMM - August and YYYY- 2021 Do will add th to the date
+moment().format('dddd');                    // Wednesday
+moment().format("MMM Do YY");               // Aug 11th 21
+console.log(moment().format('YYYY [escaped] YYYY'));     // 2021 escaped 2021
+moment().format();
+
```

Comment:
Explore utc time format.. unix format.
try performing conversions from a date in one format string to another